docker_images/bertopic/tests/test_api.py

```diff
@@ -0,0 +1,59 @@
+import os
+from typing import Dict
+from unittest import TestCase, skipIf
+
+from app.main import ALLOWED_TASKS, get_pipeline
+
+
+# Must contain at least one example of each implemented pipeline
+# Tests do not check the actual values of the model output, so small dummy
+# models are recommended for faster tests.
+TESTABLE_MODELS: Dict[str, str] = {
```

Comment:
I've updated it to parameterize inputs. I've added both cases because one model uses safetensors, and the other use Pytorch for model serialization, so I think it might be good to catch regressions in either of those scenarios.